### PR TITLE
Switch to @conda-forge/spacy team

### PR DESCRIPTION
**Pull request by adrianeboyd**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -92,7 +92,5 @@ about:
 extra:
   recipe-maintainers:
     - rmax
-    - honnibal
-    - ines
-    - adrianeboyd
     - h-vetinari
+    - conda-forge/spacy
```
Comment (h-vetinari) [inline recipe/meta.yaml:96]: With this (once rerendered) you'd remove your rights from all spacy feedstocks. This feedstock _defines_  what @conda-forge/spacy is, so you cannot use it here.

Comment (h-vetinari) [reply recipe/meta.yaml:96]: Fixed in https://github.com/conda-forge/spacy-feedstock/commit/863036591f9223a74f7022cc35272c16b688ff5a

Comment (adrianeboyd) [reply recipe/meta.yaml:96]: Oh, that makes sense, sorry for the mistake!

Comment (h-vetinari) [reply recipe/meta.yaml:96]: no worries :)

**Pull request merged**